tests/test_lib.py

```diff
@@ -2,8 +2,9 @@
 import toml
 from pathlib import Path
 import secrules_parsing
+import pytest
 
-
+@pytest.mark.skip("with semantic-release usage they will not be in sync anymore")
```

Comment:
I don't understand this change. AFAICT, the package version will be the one defined by the configuration file, which means, `version()` will return the same version number.

In any case, if they no longer match I don't see why we need to keep the test at all.

Reply:
The idea is to use the https://github.com/coreruleset/secrules_parsing/blob/master/.github/workflows/pypi-release.yml workflow for releasing. This should increment the release number and push the update to pypi, but there is no need to update the version in the local files.

Better to remove the test then.